Update note block to clarify the use of custom env-var names (#1657)
wso2/docs-choreo-dev: en/docs/develop-components/manage-component-source-configurations.md

    !!! note
        Choreo automatically generates connection configurations when you create a connection. The properties such as **name** and **resourceRef** are automatically generated. The configurations required to establish the connection will be injected into Choreo-defined environment variables.

>         If you'd like to use custom environment variable names instead of the default Choreo-defined ones, you can add the dependency in the format of a `serviceReference` under `dependencies` in the same `component.yaml v1.1` due to its backward-compatibility. Simply copy the configuration from `component.yaml v1.0` tab and paste it under `dependencies` in your `component.yaml v1.1` configuration file.

Reviewer: ```suggestion
        If you want to use custom environment variable names instead of the default Choreo-defined ones, add the dependency as a `serviceReference` under `dependencies` in the `component.yaml v1.1` to ensure backward compatibility. You can copy the configuration from the `component.yaml v1.0` tab and paste it under `dependencies` in your `component.yaml v1.1` configuration file.
```

Author: Resolved with 943716d2cdf159bdf5b32f5c2e9a99d1041da6a6
